deploy/staging/ingress.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: fdk-search-service
+spec:
+  ingressClassName: nginx
+  rules:
+    - host: fdk-search-service.staging.fellesdatakatalog.digdir.no
```

Comment:
Tror vi må høre med resten av teamet om denne, i hvert fall kurt stian

Reply:
Jepp, kan ta det opp på preik

Reply:
det beste hadde sikkert vært `search.staging...`
men det brukes allerede av fulltext, reduser denne til `fdk-search.staging...`

Reply:
Og så tar vi heller en for i bakken med kurt stian når vi nærmer oss at vi kan ta ned hele fulltext, da kan vi la denne ta over `search.staging...`

Reply:
Skal det være med (...).api.(...) ?

Reply:
ja der sier du noe, `search.api.staging...` er kanskje enda bedre, og den kan vi ha kjørende sammen med nåværende ingress som peker på fulltext

Reply:
vi tar det opp på preik :+1: